components/crowdfunding-redesign/Fundraiser.tsx

```diff
@@ -0,0 +1,301 @@
+        About
+      </Link>
+      {headingTexts.map(heading => (
+        <Link href="#" key={heading} className={linkClasses()}>
+          {heading}
+        </Link>
+      ))}
+    </div>
+  );
+};
+
+export default function Fundraiser(props) {
+  const account = props.account;
+
+  const title = account.parent ? account.name : `Support ${account.name}`;
+  const mainAccount = account.parent || account;
+  return (
+    <React.Fragment>
+      <div className="border-b bg-background">
+        <div className="mx-auto flex h-16 max-w-screen-xl items-center justify-between">
+          <Link href={`/preview/${account.parent?.slug || account.slug}`}>
+            <Avatar className="" collective={account.parent || account} />
+          </Link>
+        </div>
+      </div>
+      <div className="">
+        <div className="mx-auto max-w-screen-xl space-y-8 py-12">
+          <div className="space-y-4 text-center">
+            <h1 className="text-balance text-4xl font-semibold">{title}</h1>
+            <p className="text-lg">{account.description}</p>
+          </div>
+
+          <div className="flex grow gap-8">
+            {/* main */}
+            <div className="w-full max-w-[900px] grow space-y-4">
+              <div className="relative h-96 w-full overflow-hidden rounded-lg bg-primary/20">
+                {account.backgroundImageUrl && (
+                  <img src={account.backgroundImageUrl} alt="background" className="h-full w-full object-cover" />
+                )}
+              </div>
+
+              <p>
+                {' '}
+                {account.type === 'EVENT' ? 'Event' : 'Fundraiser'} by{' '}
+                <Link className="font-semibold text-primary hover:underline" href={`/preview/${mainAccount.slug}`}>
+                  {mainAccount.name} <ArrowRight className="inline align-middle" size={16} />
+                </Link>
+              </p>
+            </div>
+
+            {/* sidebar */}
+            <div className="w-full max-w-[420px] space-y-4">
+              <Goals account={account} />
+
+              <Button className="w-full" size="xl">
+                <FormattedMessage defaultMessage="Contribute" id="Contribute" />
+              </Button>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div className="sticky top-0 z-10 border-b border-t bg-background">
+        <div className="relative mx-auto -mb-px h-16 max-w-screen-xl">
+          <Tabs centered={false} tabs={['Fundraiser', 'Updates', 'Expenses']} />
+        </div>
+      </div>
+      <div className="flex-1 bg-background">
+        <div className="relative mx-auto grid max-w-screen-xl grid-cols-12 gap-8 py-12">
+          <div className="col-span-2">
+            <div className="sticky top-28 space-y-4">
+              <ContentOverview content={account.longDescription} />
+            </div>
+          </div>
+
+          <div className="prose prose-slate col-span-7">
+            <h2>About</h2>
```

Comment:
Missing i18n here too (and above). I'll assume we don't want to enforce i18n at this stage and won't report the other ones.

Reply:
Yep, leaving out some i18n at this stage